common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -556,4 +556,18 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
     protected String keyToString(@k@ key) {
         return @O@.toString(key);
     }
+
+    /**
+     * Fast method of finding the next power of 2 greater than or equal to the supplied value.
+     *
+     * If the value is &lt;= 0 then 1 will be returned.
+     *
+     * This method is not suitable for {@link Integer#MIN_VALUE} or numbers greater than 2^30.*
+     * @param value from which to search for next power of 2
+     * @return The next power of 2 or the value itself if it is a power of 2
+     */
+    private static int findNextPositivePowerOfTwo(final int value)
```

Comment:
I had considered that but didn't immediately find a good place. Any suggestions?